ArmA.Studio.Data/Configuration/ComboBoxItem.cs

```diff
@@ -13,15 +13,34 @@ public class ComboBoxItem<T> : PropertyItem
         private readonly static DataTemplate ThisDataTemplate = LoadFromEmbeddedResource<DataTemplate>(typeof(ComboBoxItem<T>).Assembly, "ArmA.Studio.Data.Configuration.ComboBoxItem.xaml");
         public IEnumerable<KeyValuePair<string, T>> KeyValueCollection { get; private set; }
 
-        public ComboBoxItem(IEnumerable<KeyValuePair<string, T>> values, string name, string path, object propertyOwner) : this(values, name, string.Empty, propertyOwner.GetType().GetProperty(path), propertyOwner) { }
-        public ComboBoxItem(IEnumerable<KeyValuePair<string, T>> values, string name, string icon, string path, object propertyOwner) : this(values, name, icon, propertyOwner.GetType().GetProperty(path), propertyOwner) { }
-        public ComboBoxItem(IEnumerable<KeyValuePair<string, T>> values, string name, PropertyInfo property, object propertyOwner) : this(values, name, string.Empty, property, propertyOwner) { }
+        public Func<object, object> ValueChangeCallback { get; set; } = null;
```

Comment:
it is not supposed to add value changed callbacks to Items

use property getter & setter or proxy properties for this